### ISSUE TITLE
Cannot set resource attribute for InputMapping inside Connector

### ISSUE THREAD

**Issue by MaxTru** [camunda/camunda-modeler#2007]
__Describe the Bug__

When having a Connector InputMapping with type `script` and an external script resource which follows another InputMapping type `String`, I cannot set the `resource` attribute of the type `script` mapping.

Example:
```
    <bpmn:serviceTask id="Activity_0zvwxwq" camunda:asyncBefore="true" camunda:asyncAfter="true">
      <bpmn:extensionElements>
        <camunda:connector>
          <camunda:inputOutput>
            <camunda:inputParameter name="to">${abc}</camunda:inputParameter>
            <camunda:inputParameter name="subject">
              <camunda:script scriptFormat="freemarker" resource="foobar" />
            </camunda:inputParameter>
          </camunda:inputOutput>
          <camunda:connectorId>myConnector</camunda:connectorId>
        </camunda:connector>
      </bpmn:extensionElements>
      <bpmn:incoming>Flow_14itenb</bpmn:incoming>
    </bpmn:serviceTask>
``` 

or the following gif:
![connector_bug](https://user-images.githubusercontent.com/42800119/98543648-156e0200-2293-11eb-8156-7d9129580185.gif)

__Steps to Reproduce__

1. Use the bpmn example from this issue
2. Try to update or view the `Script` field
=> You cannot see or update it

__Expected Behavior__

`Script` field is filled with value from the XML and can be edited.

__Environment__

- OS: Linux Manjaro
- Camunda Modeler Version: 4.3

This was first raised by a customer.

__Full BPMN Example__
```
<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_1880zuz" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.4.0-nightly.20201109">
  <bpmn:process id="Process_0vwve34" isExecutable="true">
    <bpmn:startEvent id="StartEvent_1">
      <bpmn:outgoing>Flow_14itenb</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:sequenceFlow id="Flow_14itenb" sourceRef="StartEvent_1" targetRef="Activity_0zvwxwq" />
    <bpmn:serviceTask id="Activity_0zvwxwq" camunda:asyncBefore="true" camunda:asyncAfter="true">
      <bpmn:extensionElements>
        <camunda:connector>
          <camunda:inputOutput>
            <camunda:inputParameter name="to">${abc}</camunda:inputParameter>
            <camunda:inputParameter name="subject">
              <camunda:script scriptFormat="freemarker" resource="foobar" />
            </camunda:inputParameter>
          </camunda:inputOutput>
          <camunda:connectorId>myConnector</camunda:connectorId>
        </camunda:connector>
      </bpmn:extensionElements>
      <bpmn:incoming>Flow_14itenb</bpmn:incoming>
    </bpmn:serviceTask>
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_0vwve34">
      <bpmndi:BPMNEdge id="Flow_14itenb_di" bpmnElement="Flow_14itenb">
        <di:waypoint x="215" y="117" />
        <di:waypoint x="270" y="117" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
        <dc:Bounds x="179" y="99" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_0u2a1d0_di" bpmnElement="Activity_0zvwxwq">
        <dc:Bounds x="270" y="77" width="100" height="80" />
      </bpmndi:BPMNShape>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
```

---
related to: https://jira.camunda.com/browse/SUPPORT-9132

**Comment by MaxTru** [comment]
Workaround avaialble: if switching the order of the two inputParamter mappings, the bug does not occur:
```
    <bpmn:serviceTask id="Activity_0zvwxwq" camunda:asyncBefore="true" camunda:asyncAfter="true">
      <bpmn:extensionElements>
        <camunda:connector>
          <camunda:inputOutput>
            <camunda:inputParameter name="subject">
              <camunda:script scriptFormat="freemarker" resource="foobar" />
            </camunda:inputParameter>
            <camunda:inputParameter name="to">${abc}</camunda:inputParameter>
          </camunda:inputOutput>
          <camunda:connectorId>myConnector</camunda:connectorId>
        </camunda:connector>
      </bpmn:extensionElements>
      <bpmn:incoming>Flow_14itenb</bpmn:incoming>
    </bpmn:serviceTask>
```

**Comment by MaxTru** [comment]
First analysis: the issue becomes first apparent in `bpmn-js-properties-panel` in `Script.js`.

`value.scriptResourceValue` is `undefined` but should not be.

IMO this takes at least half a day to analyze, add test cases and fix. Will therefore move to the backlog. Workaround is available (see my comment before)

**Comment by MaxTru** [comment]
As customer reported, issue still exists in v4.4.0. It was first introduced in v4.3.0.

**Comment by pinussilvestrus** [comment]
Fixed upstream by https://github.com/bpmn-io/bpmn-js-properties-panel/pull/401

**Comment by barmac** [comment]
Fixed via #2025
